### PR TITLE
SDIT-1384 StaffResource - Relax agency access to inactive

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: src/main/resources/db/migration/data/R__2_1__AGENCY_LOCATIONS.sql [hunk 1]
```diff
@@ -1,38 +1,20 @@
...
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG)                    VALUES ('BMI', 'BIRMINGHAM', 'INST', 'Y');
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, LONG_DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG)  VALUES ('LEI', 'LEEDS', 'HMP LEEDS', 'INST', 'Y');
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG)                    VALUES ('WAI', 'THE WEARE', 'INST', 'Y');
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG, DEACTIVATION_DATE) VALUES ('OUT', 'OUTSIDE', 'INST', 'N', TO_DATE('2017-07-27','YYYY-MM-DD'));
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG)                    VALUES ('TRN', 'TRANSFER', 'INST', 'Y');
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG)                    VALUES ('MUL', 'MUL', 'INST', 'Y');
+INSERT INTO AGENCY_LOCATIONS (AGY_LOC_ID, DESCRIPTION, AGENCY_LOCATION_TYPE, ACTIVE_FLAG)                    VALUES ('ZZGHI', 'GHOST', 'INST', 'N');
```
Comment (steverendell) [inline src/main/resources/db/migration/data/R__2_1__AGENCY_LOCATIONS.sql:8]: Just reformatted this file to make mmore readable

File: src/main/resources/db/migration/data/R__3_3__CASELOAD_AGENCY_LOCATIONS.sql [hunk 2]
```diff
@@ -1,40 +1,22 @@
...
-INSERT INTO CASELOAD_AGENCY_LOCATIONS (CASELOAD_ID, AGY_LOC_ID)
-  VALUES  ('MUL', 'LEI');
-INSERT INTO CASELOAD_AGENCY_LOCATIONS (CASELOAD_ID, AGY_LOC_ID)
-  VALUES  ('MUL', 'OUT');
-INSERT INTO CASELOAD_AGENCY_LOCATIONS (CASELOAD_ID, AGY_LOC_ID)
-  VALUES  ('MUL', 'TRN');
+INSERT INTO CASELOAD_AGENCY_LOCATIONS (CASELOAD_ID, AGY_LOC_ID) VALUES ('MUL', 'BXI');
+INSERT INTO CASELOAD_AGENCY_LOCATIONS (CASELOAD_ID, AGY_LOC_ID) VALUES  ('MUL', 'LEI');
+INSERT INTO CASELOAD_AGENCY_LOCATIONS (CASELOAD_ID, AGY_LOC_ID) VALUES  ('MUL', 'OUT');
+INSERT INTO CASELOAD_AGENCY_LOCATIONS (CASELOAD_ID, AGY_LOC_ID) VALUES  ('MUL', 'TRN');
```
Comment (steverendell) [inline src/main/resources/db/migration/data/R__3_3__CASELOAD_AGENCY_LOCATIONS.sql:22]: Just reformatted

File: src/main/resources/db/migration/nomis/data/R__4_23__USER_ACCESSIBLE_CASELOADS.sql [hunk 3]
```diff
@@ -50,6 +50,7 @@ INSERT INTO USER_ACCESSIBLE_CASELOADS (CASELOAD_ID, USERNAME, START_DATE) VALUES
 INSERT INTO USER_ACCESSIBLE_CASELOADS (CASELOAD_ID, USERNAME, START_DATE) VALUES ('NWEB', 'PRISON_COLLATOR_LOCAL', trunc(sysdate) - 24);
 INSERT INTO USER_ACCESSIBLE_CASELOADS (CASELOAD_ID, USERNAME, START_DATE) VALUES ('LEI', 'PRISON_ANALYST_LOCAL', trunc(sysdate) - 24);
 INSERT INTO USER_ACCESSIBLE_CASELOADS (CASELOAD_ID, USERNAME, START_DATE) VALUES ('NWEB', 'PRISON_ANALYST_LOCAL', trunc(sysdate) - 24);
+INSERT INTO USER_ACCESSIBLE_CASELOADS (CASELOAD_ID, USERNAME, START_DATE) VALUES ('ZZGHI', 'PRISON_ANALYST_LOCAL', trunc(sysdate) - 24);
```
Comment (steverendell) [inline src/main/resources/db/migration/nomis/data/R__4_23__USER_ACCESSIBLE_CASELOADS.sql:53]: Added the ghost prison ZZGHI to the caseload of PRISON_ANALYST_LOCAL

File: src/main/java/uk/gov/justice/hmpps/prison/service/AgencyService.java [hunk 4]
```diff
@@ -82,8 +82,8 @@
 public class AgencyService {
 
     private static final Comparator<Location> LOCATION_DESCRIPTION_COMPARATOR = Comparator.comparing(
-            Location::getDescription,
-            new AlphaNumericComparator());
+        Location::getDescription,
+        new AlphaNumericComparator());
```
Comment (steverendell) [inline src/main/java/uk/gov/justice/hmpps/prison/service/AgencyService.java:86]: some formatting -indentation seemed to be wrong

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**